src/Resources/config/doctrine-model/Route.orm.xml

```diff
@@ -6,9 +6,9 @@
         <field name="variablePattern" type="string" nullable="true"/>
         <!-- we hardcode the column name to overwrite column naming strategies as we have to define the index on the column name -->
         <field name="staticPrefix" type="string" nullable="true" column="staticPrefix"/>
-
+          
         <indexes>
-            <index name="prefix_idx" columns="staticPrefix"/>
+            <index columns="staticPrefix"/>
```

Comment:
Hm, I didn't have the time to check what name exactly is generated, I presume it could be something like tablename_columnname, at least that would make sense. 

Changelog ok like this: https://github.com/ilukac/routing-bundle/blob/support_redirect_with_orm/CHANGELOG.md ?